packages/dataviz/src/components/Tooltip/Tooltip.component.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './Tooltip.component.scss';
+
+export interface TooltipProps {
+	entries: TooltipEntry[];
+}
+function Tooltip({ entries }: TooltipProps) {
+	return (
+		<ul className={styles['dataviz-tooltip']}>
+			{entries.map(({ key, value }) => (
+				<li key={key}>
+					{key}: <span className={styles['dataviz-tooltip__value']}>{value}</span>
+				</li>
+			))}
+		</ul>
+	);
+}
```

Comment:
Based on the code it looks like a list more than a Tooltip.

Reply:
indeed, it's a tooltipcontent to share styles between D3 and recharts based charts.
I'm renaming the component